INFO     core.jenkins_utils:jenkins_utils.py:210 running cleanup
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 403 None
DEBUG    core.jenkins_utils:jenkins_utils.py:92 failed to get page on the first try, retrying after login
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /login?from=%2F HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /j_spring_security_check HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/securityRealm/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /computer/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/credentials/store/system/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /me/my-views/view/all/submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/appearance/configSubmit HTTP/1.1" 200 134
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /user/admin/security/ HTTP/1.1" 200 None
DEBUG    conftest:conftest.py:60 Argument --remote-allow-origins=* added to the chrome
DEBUG    conftest:conftest.py:60 Argument --disable-gpu added to the chrome
DEBUG    conftest:conftest.py:60 Argument --no-sandbox added to the chrome
DEBUG    conftest:conftest.py:60 Argument --disable-dev-shm-usage added to the chrome
DEBUG    conftest:conftest.py:60 Argument --headless=new added to the chrome
DEBUG    conftest:conftest.py:60 Argument --window-size=1920,1080 added to the chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 136.0.7103.113 in PATH: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 136.0.7103.113 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 136.0.7103.113
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 136.0.7103.113
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:231 Started executable: `/usr/bin/chromedriver` in a child process with pid: 17427 using 0 to output -3
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:39313
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "POST /session HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "POST /session/0d8522d03973cd8c03ca23b72afcda1e/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "GET /session/0d8522d03973cd8c03ca23b72afcda1e/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "POST /session/0d8522d03973cd8c03ca23b72afcda1e/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "POST /session/0d8522d03973cd8c03ca23b72afcda1e/element/f.0A99715973C907743BDACDBD35CF85A8.d.AC6B10E8295A65225CF3B0B98FAFA8A0.e.1/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "POST /session/0d8522d03973cd8c03ca23b72afcda1e/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "POST /session/0d8522d03973cd8c03ca23b72afcda1e/element/f.0A99715973C907743BDACDBD35CF85A8.d.AC6B10E8295A65225CF3B0B98FAFA8A0.e.3/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "POST /session/0d8522d03973cd8c03ca23b72afcda1e/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "POST /session/0d8522d03973cd8c03ca23b72afcda1e/element/f.0A99715973C907743BDACDBD35CF85A8.d.AC6B10E8295A65225CF3B0B98FAFA8A0.e.9/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "GET /session/0d8522d03973cd8c03ca23b72afcda1e/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "GET /session/0d8522d03973cd8c03ca23b72afcda1e/cookie HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 155
DEBUG    core.jenkins_utils:jenkins_utils.py:31 update_crumb: 900c330be3ecadfe3f8201fbb71d7a268097d014c458e0f20e779ea5fca56220
DEBUG    LoginPage:login_page.py:32 login crumb: 900c330be3ecadfe3f8201fbb71d7a268097d014c458e0f20e779ea5fca56220
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "GET /session/0d8522d03973cd8c03ca23b72afcda1e/window HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "POST /session/0d8522d03973cd8c03ca23b72afcda1e/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "GET /session/0d8522d03973cd8c03ca23b72afcda1e/window/handles HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "GET /session/0d8522d03973cd8c03ca23b72afcda1e/window/handles HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "POST /session/0d8522d03973cd8c03ca23b72afcda1e/window HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "GET /session/0d8522d03973cd8c03ca23b72afcda1e/title HTTP/1.1" 200 0
DEBUG    Header:ui_element.py:56 Click on locator ('xpath', '//a[@href="/logout"]')
DEBUG    Header:ui_element.py:28 waiting for element_to_be_clickable(('xpath', '//a[@href="/logout"]'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "POST /session/0d8522d03973cd8c03ca23b72afcda1e/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "POST /session/0d8522d03973cd8c03ca23b72afcda1e/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "GET /session/0d8522d03973cd8c03ca23b72afcda1e/element/f.A023A882515B750AF25747619E0332B1.d.A63C40AC78B030A539ACA5EA45731AEC.e.75/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "POST /session/0d8522d03973cd8c03ca23b72afcda1e/element/f.A023A882515B750AF25747619E0332B1.d.A63C40AC78B030A539ACA5EA45731AEC.e.75/click HTTP/1.1" 200 0
DEBUG    LoginPage:ui_element.py:28 waiting for text_to_be_present_in_element(('xpath', '//main//h1'), 'Sign in to Jenkins')
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "POST /session/0d8522d03973cd8c03ca23b72afcda1e/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "GET /session/0d8522d03973cd8c03ca23b72afcda1e/element/f.A023A882515B750AF25747619E0332B1.d.5AE109B7DFE3F8920D0B7767817C3E3E.e.8/text HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "POST /session/0d8522d03973cd8c03ca23b72afcda1e/window HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "POST /session/0d8522d03973cd8c03ca23b72afcda1e/refresh HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "GET /session/0d8522d03973cd8c03ca23b72afcda1e/title HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:39313 "DELETE /session/0d8522d03973cd8c03ca23b72afcda1e HTTP/1.1" 200 0